src/views/demoDetails/HighlightBox.tsx

```diff
@@ -137,15 +193,87 @@ function KillHighlightBox(
   }
 }
 
+function KillStreakHighlightBox(
+  highlight: KillStreakHighlight,
+  playerMap: Map<UserId, PlayerSummary>
+) {
+  const { classes } = useStyles({ justifyContent: "center" });
+  const { player, streak } = highlight;
+  let message;
+  switch (streak) {
+    case 5:
+      message = "is on a Killing Spree!";
+      break;
+    case 10:
+      message = "is Unstoppable!";
+      break;
+    case 15:
+      message = "is on a Rampage!";
+      break;
+    case 20:
+      message = "is GOD-like!";
+      break;
+    default:
+      if (streak > 0 && streak % 5 === 0) {
+        // Multiple of 5 and greater than 20, so it's a continuation of a godlike streak
+        message = "is still GOD-like!";
+      } else {
+        // Bad reporting? Fall back to reasonable message
+        message = "is collecting frags!";
+      }
+      break;
+  }
+
+  return (
+    <div className={classes.root}>
+      <span>
+        <PlayerName player={player} /> {message}
+      </span>
+      <KillstreakIcon streak={streak} />
+    </div>
+  );
+}
+
+function KillStreakEndedHighlightBox(
+  highlight: KillStreakEndedHighlight,
+  playerMap: Map<UserId, PlayerSummary>
+) {
+  const { classes } = useStyles({ justifyContent: "center" });
+  const { killer, victim, streak } = highlight;
+
+  let message;
+  if (killer.user_id === victim.user_id) {
+    message = (
+      <span>
+        <PlayerName player={killer} /> ended their own killstreak
+      </span>
+    );
+  } else {
+    message = (
+      <span>
+        <PlayerName player={killer} /> ended <PlayerName player={victim} />
+        &apos;s killstreak
+      </span>
+    );
+  }
+
+  return (
+    <div className={classes.root} style={{ display: "flex", margin: "auto" }}>
```

Comment:
I think the style prop is redundant, since `classes.root` already specifies `display:flex`, and `justifyContent: "center"` to center-align the text.
```suggestion
    <div className={classes.root}>
```